src/rpc/handlers/Ledger.cpp

```diff
@@ -79,8 +79,9 @@ LedgerHandler::process(LedgerHandler::Input input, Context const& ctx) const
                 return toJsonWithBinaryTx(tx, ctx.apiVersion);
             };
 
+            auto const isoTimeStr = ripple::to_string_iso(lgrInfo.closeTime);
```

Comment:
Could you explain what is the difference except optimisation?
`lgrInfo` is `const` so it shouldn't be different in every call of this lambda.